projects/components/src/table/table.component.ts

```diff
@@ -158,7 +158,7 @@ import { TableColumnConfigExtended, TableService } from './table.service';
             selectable: this.supportsRowSelection()
           }"
           class="data-row"
-          [style.height]="this.rowHeight"
+          [style.min-height]="this.rowHeight"
```

Comment:
I feel like I see a PR changing his back and forth between height and min height like twice a month. Please tread carefully and dig through PR history for context.

Reply:
Here's one, looks like it was probably reverted more recently - https://github.com/hypertrace/hypertrace-ui/pull/1711/files

Reply:
Here's another https://github.com/hypertrace/hypertrace-ui/pull/1724/files

Reply:
Yeah, this is an issue, but the `height` change, created more issues in the table, sometime it looks like all the tables are broken due to that.